hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OzoneFsServerDefaults.java

```diff
@@ -37,7 +37,7 @@ public OzoneFsServerDefaults() {
   }
 
   public OzoneFsServerDefaults(String keyProviderUri) {
-    super(0L, 0, 0, (short)0, 0, false, 0L, null, keyProviderUri);
+    super(0L, 512, 0, (short)0, 0, false, 0L, null, keyProviderUri);
```

Comment:
```suggestion
    super(0L, 16 * 1024, 0, (short)0, 0, false, 0L, null, keyProviderUri);
```
Thanks for filing the bug!
Ozone uses default checksum size of 16KB (previously we even had default to be 1MB)

It was updated in [HDDS-10465](https://issues.apache.org/jira/browse/HDDS-10465) by @ChenSammi

Reply:
@jojochuang Thank you for reviewing and sharing that information! I’ve committed your suggestion.